config/rbac/appstudio_pipelines_runner_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: appstudio-pipelines-runner
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: appstudio-pipelines-runner
+subjects:
+- kind: ServiceAccount
+  name: controller-manager
```

Comment:
Why do we need to create a role binding for the control manager? Why not for the member operator SA?

Can you add a detailed comment to this yaml explaining why we need this?

Reply:
Ok, I'm idiot. I didn't realize that it won't be actually deployed as it is, which means that the ClusterRole won't be available with that name :man_facepalming: :man_facepalming: :man_facepalming: